cli.js

```diff
@@ -17,6 +17,7 @@ var cli = meow({
 		'',
 		'Options',
 		'  --json     Output the result as JSON',
+		'  --bytes    Output the result in megabytes per second',
 		'  --verbose  Output more detailed information'
 	]
 });
```

Comment:
I guess it would be best practice to add a [`boolean`](https://github.com/substack/minimist#var-argv--parseargsargs-opts) property here.

```js
var cli = meow({
	// help here
}, {
	boolean: ['json', 'bytes', 'verbose']
});
```

But I'll leave it to @sindresorhus to decide if this should be added. In the next release of `meow`, it won't infer types in the by default.